### PR TITLE
Add more detail comments for AggregatedStatus field

### PR DESCRIPTION
**Pull request by mkloveyy**
Signed-off-by: mkloveyy [halfpig0627@163.com](mailto:halfpig0627@163.com)

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
It add more comments for AggregatedStatus definition.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/config/v1alpha1/interpretercontext_types.go [hunk 1]
```diff
@@ -62,7 +62,8 @@ type ResourceInterpreterRequest struct {
 	DesiredReplicas *int32 `json:"replicas,omitempty"`
 
 	// AggregatedStatus represents status list of the resource running in each member cluster.
-	// +optional
+        // It'll be set only if InterpreterOperation is InterpreterOperationAggregateStatus.
```
Comment (RainbowMango) [inline pkg/apis/config/v1alpha1/interpretercontext_types.go:65]: Actually, it always is set, no matter [AggregateStatus](https://karmada.io/docs/userguide/globalview/customizing-resource-interpreter/#aggregatestatus) is enabled or not for the resource.

Comment (RainbowMango) [reply pkg/apis/config/v1alpha1/interpretercontext_types.go:65]: Oh, sorry, my bad. You are right! Ignore me.

I thought the comments is added to here:
https://github.com/karmada-io/karmada/blob/4011f07178c97648ce547640f04d240dd38d8004/pkg/apis/work/v1alpha2/binding_types.go#L253-L255

**Comment by RainbowMango** [comment]
Hi @mkloveyy , Could you please add your email to [GitHub Emails](https://github.com/settings/emails)? Otherwise, GitHub can't recognize you and count your contributions.

**Comment by mkloveyy** [comment]
@RainbowMango I'm sorry. Now I've changed the email in commit message. Is that worked?

**Comment by RainbowMango** [comment]
Yes, it worked. 
image

**Comment by chaunceyjiang** [comment]
Please exec the shell script ` hack/update-all.sh`.

**Comment by mkloveyy** [comment]
@chaunceyjiang I've execute but nothing changed and I format the file and commit again.

**Comment by RainbowMango** [comment]
> @chaunceyjiang I've execute but nothing changed and I format the file and commit again.

The gofmt issue has been resolved.

You can run `make update`(or ` hack/update-all.sh`) at the `root of the repo`, like:
```
# make update
...
# git status
On branch review343101
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pkg/generated/openapi/zz_generated.openapi.go    // this is the outdate file

no changes added to commit (use "git add" and/or "git commit -a")
```

**Comment by mkloveyy** [comment]
> You can run `make update`(or ` hack/update-all.sh`) at the `root of the repo`, like:
> 
> ```
> # make update
> ...
> # git status
> On branch review343101
> Changes not staged for commit:
>   (use "git add <file>..." to update what will be committed)
>   (use "git restore <file>..." to discard changes in working directory)
> 	modified:   pkg/generated/openapi/zz_generated.openapi.go    // this is the outdate file
> 
> no changes added to commit (use "git add" and/or "git commit -a")
> ```

Sorry for getting back to you late. I've committed again.

**Comment by RainbowMango** [comment]
> I've executed hack/update-codegen.sh several times but nothing happened. It ends with:

Not `hack/update-codegen.sh`, it's the `hack/update-all.sh` or `#make update`.

**Comment by RainbowMango** [comment]
Sorry, I just realized that it is [hack/update-codegen.sh](https://github.com/karmada-io/karmada/blob/effd5f8f81d1d2afce4b7f425f5120a0c956444e/hack/update-codegen.sh#L187-L201) generates the open-api files.

Can you try to clone Karmada into $GOPATH? I'm not sure if it will work.

**Comment by RainbowMango** [comment]
Congratulations seem you made it!
How did you work it out?

Just one thing remaining: Could you please [squash your commits](https://karmada.io/docs/contributor/github-workflow#squash-commits)? Then everything will be all set.

**Comment by mkloveyy** [comment]
> Congratulations seem you made it! How did you work it out?

Sry. I tried but still not work, so I edit the `pkg/generated/openapi/zz_generated.openapi.go` temporarily to make it passed.
Have you ever met same scenario before?

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Comment by RainbowMango** [comment]
> Have you ever met same scenario before?

I'm not sure.  A long time ago, the code generator requires the Karmada repo must be cloned at the $GOPATH, but this restriction has been optimized.

**Pull request merged**